test/metric_value_benchmark/mem_test.go

```diff
@@ -10,41 +10,49 @@ import (
 	"time"
 
 	"github.com/aws/amazon-cloudwatch-agent-test/test/metric"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
+
 	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
+	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
 )
 
 type MemTestRunner struct {
-	BaseTestRunner
+	test_runner.BaseTestRunner
+	Base test_runner.BaseTestRunner
 }
 
-var _ ITestRunner = (*MemTestRunner)(nil)
+var _ test_runner.ITestRunner = (*MemTestRunner)(nil)
 
-func (m *MemTestRunner) validate() status.TestGroupResult {
-	metricsToFetch := m.getMeasuredMetrics()
+func (m *MemTestRunner) Validate() status.TestGroupResult {
+	metricsToFetch := m.GetMeasuredMetrics()
 	testResults := make([]status.TestResult, len(metricsToFetch))
 	for i, name := range metricsToFetch {
 		testResults[i] = m.validateMemMetric(name)
 	}
 
 	return status.TestGroupResult{
-		Name:        m.getTestName(),
+		Name:        m.GetTestName(),
 		TestResults: testResults,
 	}
 }
 
-func (m *MemTestRunner) getTestName() string {
+func (m *MemTestRunner) GetTestName() string {
 	return "Mem"
 }
 
-func (m *MemTestRunner) getAgentConfigFileName() string {
+func (m *MemTestRunner) GetAgentConfigFileName() string {
 	return "mem_config.json"
 }
 
-func (m *MemTestRunner) getAgentRunDuration() time.Duration {
-	return minimumAgentRuntime
+func (m *MemTestRunner) GetAgentRunDuration() time.Duration {
+	return test_runner.MinimumAgentRuntime
+}
+
+func (t *MemTestRunner) SetupAfterAgentRun() error {
```

Comment:
Dont think we need these overrides since returning nil is the default behavior. In fact it might be nice to make the above `GetAgentRunDuration` a default on `BaseTestRunner` as well

Reply:
yes, I forgot to delete them. this was before I realized I missed that was possible. had build breaks that was fixable without doing this

Reply:
For some of these things I feel like there should be a better way - because right now this is kinda technically has problems of inheritance. But not something I wanna worry about rn